app/classifier/tasks/survey/editor.cjsx

```diff
@@ -14,97 +14,184 @@ Papa = require 'papaparse'
-            <i className="fa fa-question-circle"></i>
-          </span>
-        }>
-          {surveyEditorHelp.characteristics}
-        </TriggeredModalForm>
-      </div>
-      <div className="columns-container" style={marginBottom: '0.2em'}>
-        <FileButton className="major-button column" accept=".csv, .tsv" multiple onSelect={@handleFiles.bind this, @addConfusion, null}>Add confused pairs CSV</FileButton>
-        <TriggeredModalForm trigger={
-          <span className="secret-button">
-            <i className="fa fa-question-circle"></i>
-          </span>
-        }>
-          {surveyEditorHelp.confusions}
-        </TriggeredModalForm>
-      </div>
-      <div className="columns-container" style={marginBottom: '0.2em'}>
-        <FileButton className="major-button column" multiple onSelect={@handleFiles.bind this, @addQuestion, @cleanQuestions}>Add questions CSV</FileButton>
-        <TriggeredModalForm trigger={
-          <span className="secret-button">
-            <i className="fa fa-question-circle"></i>
-          </span>
-        }>
-          {surveyEditorHelp.questions}
-        </TriggeredModalForm>
-      </div>
-
+      <span className="form-label">INSTRUCTIONS</span>
+      <p><small>The survey task is ideal for asking volunteers to identify wildlife species in camera trap photos, but can be used anytime you have a lot of choices you need volunteers to filter from.</small></p>
+      <p><small><a href="https://www.zooniverse.org/projects/aliburchard/cameratraptest/about/faq">See here</a> for detailed instructions.</small></p>
+      <p><small><a href="http://bit.ly/1QSpevJ">Click here</a> to download Choices, Characteristics, Confusions and Questions templates.</small></p>
+      <p><small>You can Apply changes after adding an individual csv file or many csv files.</small></p>
       <hr />
 
-      {unless @state.importErrors.length is 0
-        <div>
-          <p className="form-label">Import errors</p>
-          {for {error, file, row}, i in @state.importErrors
-            <p key={i}>
-              <strong>{error.message}</strong>
-              <br />
-              <span className="form-help">
-                {if file?
-                  <span>{file?.name} </span>}
-                {if row? or error.row?
-                  <span>Row {(row ? error.row) + 1} </span>}
-              </span>
-            </p>}
-          <hr />
-        </div>}
-
-      <div>
-        <span className="form-label">Survey images</span>{' '}
-        <AutoSave resource={@props.workflow}>
-          <button type="button" disabled={@state.resettingMedia} onClick={@resetMedia}>Delete all</button>
-          {if @state.resettingMedia
-            <i className="fa fa-spinner fa-spin"></i>}
-        </AutoSave>
+      <p>
+        <span className="form-label">FILE INPUT</span>
+        {unless @checkApply()
+          <span className="form-label warning"> ** WARNING: unapplied content **</span>}
```

Comment:
Have we had Jordan take a look at this design? I feel like we can, ultimately, provide a better user experience than a warning that is present whenever changes are made, given that we expect people to make changes. (not necessary for merge)